### PR TITLE
Some changes needed for Zoom plugin

### PR DESCRIPTION
**Pull request by pferreir**
This adds:
 * Possibility to disable WTFPrincipalField;
 * Template hook to add additional interface items in list of VC rooms;
 * New `is_new` flag in `update_data_vc_room`.

### CODE REVIEW

File: indico/web/templates/forms/principal_widget.html [hunk 1]
```diff
@@ -13,7 +13,8 @@
         setupPrincipalWidget({
             fieldId: {{ field.id | tojson }},
             required: {{ input_args.required | default(false) | tojson }},
-            withExternalUsers: {{ field.allow_external_users | tojson }}
+            withExternalUsers: {{ field.allow_external_users | tojson }},
+            disabled: {{ field.render_kw.disabled if field.render_kw else None | tojson }}
```
Comment (ThiefMaster) [inline indico/web/templates/forms/principal_widget.html:17]:
```suggestion
            disabled: {{ (field.render_kw.disabled if field.render_kw else none) | tojson }}
```

File: indico/modules/vc/plugins.py [hunk 2]
```diff
@@ -185,7 +185,7 @@ def update_data_association(self, event, vc_room, event_vc_room, data):
         if event_vc_room.data is None:
             event_vc_room.data = {}
 
-    def update_data_vc_room(self, vc_room, data):
+    def update_data_vc_room(self, vc_room, data, is_new=False):
```
Comment (ThiefMaster) [inline indico/modules/vc/plugins.py:188]: :warning: This requires a change in the `vc_vidyo` plugin as well, unless we call it with `is_new=True` only if the method accepts this argument.

Comment (pferreir) [reply indico/modules/vc/plugins.py:188]: Indeed!

Comment (pferreir) [reply indico/modules/vc/plugins.py:188]: Solved, thanks!

**Pull request merged**